.gitignore

```diff
@@ -1,28 +1,20 @@
-*~
-.idea/
-.php_cs.cache
-.svn/
-/_build/
-/app.yaml
-/backup/
-/composer.phar
-/config/config.local.php
-/config/queue.local.php
-/config/*.pem
-/log/
-/nbproject/private/
+.idea
 /node_modules
-/public/js/1stparty.min.js
-/public/js/1stparty.min.js.map
-/public/js/3rdparty.min.js
-/public/js/3rdparty.min.js.map
-/public/js/CDash_*.min.js*
```

Comment:
I think we should continue to ignore these transitory build artifacts. Any reason not to?

Reply:
Nevermind, I see that the old .gitignore file still exists (and is functional) in a subdirectory.